Patch 1

wth461694678 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/wth461694678/text-block-timer

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

wth461694678 commented:
The main.js file has already been removed from the repo with the .gitignore updated.
In additional, a trivial typing error has been corrected in README.md.

joethei commented:
Where is your source code?
If you have written your plugin in plain JavaScript then the bot message can be ignored, that's a false positive.

wth461694678 commented:
> Where is your source code? If you have written your plugin in plain JavaScript then the bot message can be ignored, that's a false positive.

Hi joethei, I've added the source code to the repository and added an English version of the README.
